samples/bash/challenge/image/Dockerfile

```diff
@@ -18,7 +18,6 @@ COPY nsjail.cfg /home/user/
 
 CMD kctf_setup && \
     drop_privs \
-    nsjail --config /kctf/nsjail_base.cfg --config /home/user/nsjail.cfg -- \
-    /usr/bin/stdbuf -i0 -o0 -e0 \
-    maybe_pow.sh \
-    /bin/bash
+    socat \
+      TCP-LISTEN:1337,reuseaddr,fork \
+      EXEC:"/usr/bin/stdbuf -i0 -o0 -e0 maybe_pow.sh nsjail --config /home/user/nsjail.cfg -- /bin/bash"
```

Comment:
Can we make this a shell script? That just accepts params

Reply:
And ship it as part of the nsjail image? (we should btw rename that image, kctf-base or something)

I'm torn. I think it would look nicer but it also hides the details how this works and it might prevent challenge authors to make changes (maybe the script will do other required stuff that they don't know or start doing it in the future).

Reply:
Sent another suggestion below. Not blocking though.